### PR TITLE
Update 'Linked accounts' to include Google Ads

### LINKED ISSUE #2541: Update the copy on the settings screen to include the Google Ads account
On the settings tab of the Google for WooCommerce page, users can disconnect all accounts altogether. The copy says: “A WordPress.com account, Google account, and Google Merchant Center account are required to use this extension in WooCommerce.” See screenshot below.

image

The copy should be updated to include Google Ads.

## Acceptance Criteria

- The linked accounts section description reads "A WordPress.com account, Google account, Google Merchant Center account, and Google Ads account are required to use this extension in WooCommerce."

## Implementation Brief

The file that needs to be edited is `js/src/settings/linked-accounts-section-wrapper.js`.

### PR DESCRIPTION
**Pull request by kt-12**
### Changes proposed in this Pull Request:

Closes #2541.

Google Ads have been added to the description.

### Screenshots:

Before: 
Text: **A WordPress.com account, Google account, and Google Merchant Center account are required to use this extension in WooCommerce.**
image

After: 
Text: **A WordPress.com account, Google account, Google Merchant Center account, and Google Ads account are required to use this extension in WooCommerce.**
Screenshot 2024-08-22 at 12 26 05

### Detailed test instructions:

1. Start onboarding process 1st step. Under `Linked Accounts` check the text.

### Changelog entry

> Tweak - Update the copy in the "Linked accounts" of the accounts connection setting to include Google Ads account.

### CODE REVIEW

**Review by joemcgill — APPROVED** [review]
🎯 This one is fine to merge directly to `develop` once approved.

**Review by ankitguptaindia — APPROVED** [review]
The text has been updated. ✅ 

| BEFORE | AFTER |
|--------|--------|
| image |image|

**Review by eason9487 — APPROVED** [review]

**Comment by joemcgill** [comment]
@eason9487 I don't have permissions to merge to the `develop` branch. Can you merge this one? I think it's fine to go in early.

**Pull request merged**
